angularjs-1.5-basics/src/app/landing-page/common/convertToHtml.filter.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
__[jshint]__
* Use the function form of "use strict".

<!-- Comment by [SideCI](https://sideci.com) -->